templates/error.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>Build error</title>
+    <script src="/ember-cli-live-reload.js" type="text/javascript"></script>
```

Comment:
this is a hack, but a simple way to get live-reload from the containing server.

We could make this configurable, or.... maybe even better just make the whole template configurable. This would allow easy customization